pkg/k8shandler/curation.go

```diff
@@ -55,7 +55,8 @@ func (clusterRequest *ClusterLoggingRequest) CreateOrUpdateCuration() (err error
 
 		printUpdateMessage := true
 		retryErr := retry.RetryOnConflict(retry.DefaultRetry, func() error {
-			if !reflect.DeepEqual(curatorStatus, cluster.Status.Curation.CuratorStatus) {
+
+			if !compareCuratorStatus(curatorStatus, cluster.Status.Curation.CuratorStatus) {
```

Comment:
Same question here: why do we need to compare status?  Who else is updating it that we should get a conflict?  @bparees should i expect a conflict?

Reply:
its not about preventing conflicts, its about not making unnecessary calls. If the status is exactly the same as what is out there, there is nothing to actually update so we want to spare k8s needing to process our update